libraries/GCS_MAVLink/MAVLink_routing.cpp

```diff
@@ -90,6 +93,15 @@ routing table.
 */
 bool MAVLink_routing::check_and_forward(GCS_MAVLINK &in_link, const mavlink_message_t &msg)
 {
+    #if HAL_SOLO_GIMBAL_ENABLED
+    // check if a Gopro is connected. If yes, we allow the routing
+    // of mavlink messages to a private channel (Solo Gimbal case)
+    if ((gopro_status_check == false) && (msg.msgid == MAVLINK_MSG_ID_GOPRO_HEARTBEAT)) {
```

Comment:
nitpick: we could change, "gopro_status_check == false" to "!gopro_status_check"